bin/build.sh

```diff
@@ -388,25 +409,26 @@ then
 
   puts "Configuring shell environment"
   echo export BOOGIE=\"mono ${BOOGIE_DIR}/Binaries/Boogie.exe\" >> ${SMACKENV}
-  echo export CORRAL=\"mono ${CORRAL_DIR}/bin/Release/corral.exe\" >> ${SMACKENV}
-  echo export LOCKPWN=\"mono ${LOCKPWN_DIR}/Binaries/lockpwn.exe\" >> ${SMACKENV}
-  source ${SMACKENV}
-  puts "The required environment variables have been set in ${SMACKENV}"
-  puts "You should source ${SMACKENV} in your .bashrc"
-
-  puts "Built SMACK"
+	echo export CORRAL=\"mono ${CORRAL_DIR}/bin/Release/corral.exe\" >> ${SMACKENV}
```

Comment:
It seems that indentation got messed up here. Maybe you are using tabs instead of spaces? Not sure, but please take a look.